browser/html/framed.doc.html

```diff
@@ -300,7 +318,9 @@
         }
 
         let messageOut = document.getElementById("messages");
-        messageOut.textContent = messageOut.textContent + JSON.stringify(msg) + "\n";
+        let line = document.createElement("p");
```

Comment:
```suggestion
        let line = document.createElement("pre");
```
Using <pre> should allow not to use `font-family: monospace`

Reply:
but using pre would make everything overflow impossible to wrap.

Reply:
Using p with monospace is fine.